src/main/resources/db-scripts/migration.sql

```diff
@@ -1024,3 +1024,44 @@ CREATE INDEX idx_clinical_event_key ON clinical_event_data (`KEY`);
 CREATE INDEX idx_clinical_event_value ON clinical_event_data (`VALUE`);
 CREATE INDEX idx_sample_stable_id ON sample (`STABLE_ID`);
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.13.1";
+
+##version: 2.14.0
+
+-- Step 1: Drop all foreign keys
+SET FOREIGN_KEY_CHECKS=0;
+
+-- CREATE PROCEDURE DROP_FOREIGN_KEY_IF_EXISTS(IN tableName VARCHAR(64), IN constraintName VARCHAR(64))
+--     BEGIN
+--         IF EXISTS(
+--             SELECT * FROM information_schema.table_constraints
+--             WHERE 
+--                 table_schema    = DATABASE()     AND
+--                 table_name      = tableName      AND
+--                 constraint_name = constraintName AND
+--                 constraint_type = 'FOREIGN KEY')
+--         THEN
+--             SET @query = CONCAT('ALTER TABLE ', tableName, ' DROP FOREIGN KEY ', constraintName, ';');
+--             PREPARE stmt FROM @query; 
+--             EXECUTE stmt; 
+--             DEALLOCATE PREPARE stmt; 
+--         END IF; 
+--     END
+
+DROP PROCEDURE IF EXISTS DROP_FOREIGN_KEY_IF_EXISTS;
+-- This all has to be on a single line for the migrate_db script to parse it correctly
+CREATE PROCEDURE DROP_FOREIGN_KEY_IF_EXISTS(IN tableName VARCHAR(64), IN constraintName VARCHAR(64)) BEGIN IF EXISTS (SELECT 1 FROM information_schema.table_constraints WHERE table_schema = DATABASE() AND table_name = tableName AND constraint_name = constraintName AND constraint_type = 'FOREIGN KEY') THEN SET @query = CONCAT('ALTER TABLE ', tableName, ' DROP FOREIGN KEY ', constraintName); PREPARE stmt FROM @query; EXECUTE stmt; DEALLOCATE PREPARE stmt; END IF; END;
+
+CALL DROP_FOREIGN_KEY_IF_EXISTS('clinical_event_data', 'clinical_event_data_ibfk_1');
+
```

Comment:
Now that we have executed the stored procedure I think it would make sense to clean it up so that it is not left on the MYQSL system without a known future purpose. So I suggest adding another statement here for:
DROP PROCEDURE IF EXISTS DROP_FOREIGN_KEY_IF_EXISTS;

Reply:
Sure